docs/DeveloperGuide.md

```diff
@@ -1,38 +1,269 @@
-# Developer Guide
+<div align="center">
+<h2> MoneyMind Developer Guide</h2>
+<h3> "Mind your Money" </h3>
+</div>
+
+<div align="left">
 
```

Comment:
For some reason, the formatting goes haywire from this point onwards.
![image](https://user-images.githubusercontent.com/88264296/228424204-b3245139-e7fb-48a9-8778-14f38a55fa24.png)